scripts/entity/components/attack_components/player_attack_component.gd

```diff
@@ -23,7 +23,7 @@ func _ready() -> void:
 	# Abstract class instantiation error
 	lock_on_area.body_entered.connect(_add_target)
 	lock_on_area.body_exited.connect(_remove_target)
-	on_target_change.connect(target_changed)
+	# on_target_change.connect(target_changed)
```

Comment:
If you don't need the code I would delete it rather than uncomment it. If you do want to keep it you should put a comment saying why it is commented out otherwise it can only cause confusion on why it is commented out.